utils/pattern/src/double.rs

```diff
@@ -234,14 +234,6 @@ impl DoublePlaceholderInfo {
 ///     "yesterday",
 /// );
 ///
-/// // Escaped placeholder and a placeholder value 1 (note, "bar" is used):
```

Comment:
this test is being added back in #5531, but there's no way to pass options through `FromStr`.